test/integration/includes.cc

```diff
@@ -318,3 +318,29 @@ TEST(IncludesTest, Includes_15_convert)
   EXPECT_EQ("1.6", modelElem->OriginalVersion());
   EXPECT_EQ("1.6", linkElem->OriginalVersion());
 }
+
+//////////////////////////////////////////////////
+TEST(IncludesTest, IncludeModelMissingConfig)
+{
+  sdf::setFindCallback(findFileCb);
+
+  std::ostringstream stream;
+  stream
+    << "<sdf version='" << SDF_VERSION << "'>"
+    << "<include>"
+    << "  <uri>box_missing_config</uri>"
+    << "</include>"
+    << "</sdf>";
+
+  sdf::SDFPtr sdfParsed(new sdf::SDF());
+  sdf::init(sdfParsed);
+  ASSERT_TRUE(sdf::readString(stream.str(), sdfParsed));
+
+  sdf::Root root;
+  sdf::Errors errors = root.Load(sdfParsed);
+  for (auto e : errors)
+    std::cout << e.Message() << std::endl;
+  EXPECT_TRUE(errors.empty());
```

Comment:
How about checking the error code and the exact number of errors?

Reply:
the error was actually coming from `sdf::readString`, because I was using an overload of `sdf::readString` that printed `sdf::Errors` instead of returning them. I've switched APIs and added expectations for the error type and message in 5bf57af2a6b00d3de29fba9bea4858ea54177e92.